content/projects/memory-game/part-2/_index.md

```diff
@@ -17,6 +17,10 @@ ready: true
 submission_type: continue_repo
 title: extra features for your memory game
 weight: 4
+tags:
+- TDD
+- Dom Manipulation
```

Comment:
It's best to stick to one naming convention
Probably lowercase stuff with dashes is the way to go. 

so instead of TDD, tdd
instead of Dom Manipulation, dom-manipulation